dev-util/mh/files/Makefile.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/Makefile b/Makefile
+index 01023ca..f87ab56 100644
+--- a/Makefile
++++ b/Makefile
+@@ -14,6 +14,7 @@ endif
+ 
+ DEBUG = -g -fsanitize=address
+ CFLAGS = -Wall -lpcre2-8 #$(DEBUG)
++LDFLAGS ?=
```

Comment:
omit this definition here, adds no value

Reply:
Removed.